release-team/role-handbooks/ci-signal/upgrade-job-versions.md

```diff
@@ -0,0 +1,154 @@
+# Upgrade Job Versions
+
+An example of how, given an upgrade job, the different ways I can verify:
+- what version is it upgrading FROM (**old-version**)
+- what version is it upgrading TO (**new-version**)
+
+## TL;DR
+- there are [very few jobs](#jobs-that-do-use-release-builds) that actually use released/tagged builds
+- most jobs use one of these versions, built off the HEAD of a branch
+
+| name           | branch eg    | actual eg                           |
+| -------------- | ------------ | ----------------------------------- |
+| ci/latest      | master       | v1.13.0-alpha.0.1226+426ef9d349bb3a |
+| ci/k8s-beta    | release-1.12 | v1.12.0-beta.1.129+1d58f1aebfe1e3   |
+| ci/k8s-stable1 | release-1.11 | v1.11.3-beta.0.71+a4529464e4629c    |
+| ci/k8s-stable2 | release-1.10 | v1.10.8-beta.0.33+a963fce72fed31    |
+| ci/k8s-stable3 | release-1.9  | v1.9.11-beta.0.36+06bf888123ca89    |
+
+NB: these examples are based on the current cycle at time of writing, which is release-1.12
+
+### Option A: Look at the job name
+aka trust the job
+
+ref: https://github.com/kubernetes/community/blob/master/contributors/devel/e2e-tests.md#test-jobs-naming-convention
+
+- upgrade-cluster: master + nodes upgraded to **new-version**, **old-version** tests run
+- upgrade-cluster-new: master + nodes upgraded to **new-version**, **new-version** tests run
+- upgrade-master: master upgraded to **new-version**, nodes left at **old-version**, **old-version** tests run
```

Comment:
Covered explicitly later, but would it make sense here to have one more bullet to given an example of a name that can be trusted to mean what's under test is "stable" (ie: tagged, released version not master), like the "stable-1-y" in:
`ci-kubernetes-gce-conformance-stable-1-y`

Reply:
I don't actually have that level of trust at the moment until we more fully audit job names, as you've noted, they're not quite right